apps/web/src/studio/utils/routing.ts

```diff
@@ -8,10 +8,6 @@ export function isStudioHome(pathname: string) {
   return matchPath(STUDIO_WORKFLOWS_HOME_ROUTE, pathname);
 }
 
-export function isStudioRoute(path: string) {
-  return path.includes('/studio');
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Why remove this? This centralizes the implementation without being dependent on data source?‏

Reply:
When using react router the routing management is part of the rendering react cycle. That is routing is handled mostly by hooks. We can have utilities functions when it makes sense but we will need to use them from hooks. For one liners like this, defining them in the hook seems easier to avoid another import.

Moreover, it's about the "framework" we want to establish in the SPA. My goal is to try to keep it as simple as it gets. Since we don't use any meta framework yet, in a React SPA the expectation is usually to see components, providers, and hooks. Utils and helpers might be present as well, but I'd rather minimize their usage. 

I acknowledge this is a very nit-picky comment. We can get back to it later. If you feel strong I can revert it and use the utility from the hook.